src/components/App/App.js

```diff
@@ -1,10 +1,90 @@
 import styled from 'styled-components/macro'
 import Countdown from '../Countdown/Countdown'
+import { ReactComponent as PlayButton } from '../../assets/play-icon.svg'
+import { ReactComponent as StopButton } from '../../assets/stop-icon.svg'
+import { useState, useEffect } from 'react'
 
 function App() {
+  const LENGTHTWENTYFIVE = { minutes: 25, seconds: 0 }
+  const LENGTHFIFTY = { minutes: 50, seconds: 0 }
+
+  const [lengthFifty, setLengthFifty] = useState(false)
```

Comment:
Suggestion:
In case you ever want a full hour or something else entirely, maybe change the variable name to something like `lengthLong`?

Reply:
Great idea, thanks!